README.md

```diff
@@ -3,72 +3,17 @@ MinIO Kubernetes Cloud
 
 ## Installation
```

Comment:
Add a line for make yamlgen?

Reply:
that's only for when we want a new yaml for operator to be checked in